src/services/ui/src/features/package/package-details/index.tsx

```diff
@@ -45,15 +45,9 @@ export const PackageDetails: FC<{
   title: string;
 }> = (props) => {
   const { data } = usePackageDetailsCache();
-  // const title = props.title || `${data.authority} Package Details`;
 
-  // andies thoughts:
-  //     abstract out the grid container from the Details Item Grid components
-  //     instead make the items just the "cells in the grid"
-  //     add some sort of logic for the description to conditionally make it take up 2 cells
-  //    ^ if it takes up 2 cells it must be on its own row
```

Comment:
@andieswift Are these comments still relevant? They could be outdated, so I removed them. Please confirm 😄

Reply:
You can remove. I believe at the time we were having a discussion around how we could refactor the details section component to make it more responsive.